circuits/sybil/sybilUtils.circom

```diff
@@ -10,7 +10,7 @@ template GetStateCommitmentSchemaHash(){
     schemaHash <== 7082351036644153942426544930816155573;
 }
 
-template VerifyAndExtractValStateCommitment(HolderLevel, GistLevels){ // stateCommitmentClaim
+template VerifyStateCommitment(HolderLevel, GistLevels){ // stateCommitmentClaim
```

Comment:
Rename `HolderLevel` to `UserLevels` for consistency. We use **user** but not **holder** naming everywhere.
Please, check for the same everywhere in the code

Reply:
Done